variants/heltec_wireless_stick_lite_v3/pins_arduino.h

```diff
@@ -0,0 +1,60 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define Wireless_Stick_Lite true
```

Comment:
Please use a different macro name so we can distinguish this from V1. I would suggest
`#define Wireless_Stick_Lite_v3 true`

Reply:
To my understanding, four main variants of the esp32/lora heltec boards exist, with I believe, three revisions each. While I understand the reasoning behind placing the version in the definition, this attribute is mainly used with their library (https://github.com/HelTecAutomation/Heltec_ESP32).

This definition allows the library to know what features are available in the board (big screen, small screen, no screen), allow external voltage, etc. (See code: https://github.com/HelTecAutomation/Heltec_ESP32/blob/master/src/heltec.cpp).

Since there is no "board hardware difference" between versions 1,2,3, placing this definition allows the library to work as expected and without depending on that library to update (since it hasn't updated for some revisions).

Reply:
The `Wireless_Stick_Lite` and the `Wireless_Stick_Lite_V3` are extremely different board hardware. The only thing similar is the form factor. The V3 uses a new ESP32-S3 cpu, different lora module, totally different pin numbers, and even different numbers of pins.

I understand you're trying to make this work with the heltec library, but please don't use a duplicate board macro here just to make an external library work. I have a [PR](https://github.com/HelTecAutomation/Heltec_ESP32/pull/100) open in that repo to support newer boards. You can also fork that repo and use your own version of the heltec library. Until they merge my PR there, I've been pointing PlatformIO at my own fork:
```
lib_deps = https://github.com/platypii/Heltec_ESP32.git
```

Reply:
I do agree in your point since editing the repo is the actual solution and not doing it half way as I was attempting.

I just saw you PR and I believe it is a great idea to actually separate the features from the boards. Solves current issues and make it easier to add new boards in the future. 

I hope it is accepted promptly.